src/snyk/common/configuration/configuration.ts

```diff
@@ -223,23 +218,16 @@ export class Configuration implements IConfiguration {
     return `${hostnameParts[2]}.${hostnameParts[3]}`.includes('snykgov.io');
   }
 
-  get snykOssApiEndpoint(): string {
+  get snykApiEndpoint(): string {
     if (this.customEndpoint) {
-      return this.customEndpoint; // E.g. https://app.eu.snyk.io/api
+      return this.customEndpoint + '/v1';
```

Comment:
Let's also remove the /api path component, as it's not valid for api.xxx.snyk.io

Reply:
The comment in the code was wrong as the custom endpoint will always be `https://api.eu.snyk.io` now that we enforce it in the Settings UI. It seems when we were providing the custom endpoint we were always using `https://api.eu.snyk.io` so even though I originally wanted to make this `https://api.eu.snyk.io/v1` to be consistent with the `defaultApiEndpoint`, I chose instead to change `defaultApiEndpoint` to be `https://api.snyk.io` instead of `https://api.snyk.io/v1`.

This seems to work with all versions of the LS but like you said in your comment it's a bit scary because of the IntelliJ problems we've had so it might be good to have two eyes on this together?